### PR TITLE
Fix/testing doc

### PR DESCRIPTION
**Pull request by samajammin**
# Description

Update headers of Testing docs page

## Related issue(s)

None

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://github.com/privacy-scaling-explorations/maci/blob/dev/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/maci/blob/dev/CODE_OF_CONDUCT.md).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/versioned_docs/version-v1.x/testing.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ You can run them with:
 pnpm run test
 ```
 
-## Pre-Compiled Artifacts for testing
+## Pre-Compiled Artifacts for testing
```
Comment (samajammin) [inline website/versioned_docs/version-v1.x/testing.md:172]: Not sure why this fixes it but previously wasn't working:
Image 2024-01-13 at 13 39 47

File: website/versioned_docs/version-v1.x/testing.md [hunk 2]
```diff
@@ -381,7 +381,7 @@ zkeys/TallyVotes_7-3-4_test
 - [ProcessMessages_7-7-3-3_test.0.zkey](#ProcessMessages_7-7-3-3_test0zkey)
 - [TallyVotes_7-3-3_test.0.zkey](#TallyVotes_7-3-3_test0zkey)
 
-###### ProcessMessages_4-6-3-4_test.0.zkey
+#### ProcessMessages_4-6-3-4_test.0.zkey
```
Comment (samajammin) [inline website/versioned_docs/version-v1.x/testing.md:384]: Not sure if these headers are supposed to be under `Contribution Hash`? Or Standalone...

Comment (ctrlc03) [reply website/versioned_docs/version-v1.x/testing.md:384]: yes, they should be under contribution hash, as they are the contribution hashes for each circuit listed there

**Review by ctrlc03 — APPROVED** [review]
Nice, LGTM

**Pull request merged**